src/content/ui-api/management-api/crd/appcatalogentries.application.giantswarm.io.md

```diff
@@ -1,15 +1,31 @@
 ---
-title: AppCatalogEntry CRD schema reference
+title: AppCatalogEntry CRD schema reference (group application.giantswarm.io)
```

Comment:
Adding the group name to the page title, to make CRD pages with identical short names easier to distinguish from search results.